.github/workflows/build_hub.yml

```diff
@@ -116,6 +126,12 @@ jobs:
           export $(xargs < .vault/env/.env.local)
           ./prime create-credential --type=UserPass --persist=DEFAULT-SFTP --user foo --pass pass
           ./gradlew testSmoke
+          
```

Comment:
is this for debugging or meant to be permanent? (just curious, I don't think either is incorrect/wrong)

Reply:
Permanent.  This will make it easier to see why something fails.